app/src/main/java/org/simple/clinic/activity/ActivityLifecycle.kt

```diff
@@ -4,25 +4,25 @@ import org.simple.clinic.widgets.UiEvent
 
 private fun event(activityName: String?, event: String): String = activityName?.let { "$activityName:$event" } ?: ""
 
-sealed class TheActivityLifecycle : UiEvent {
+sealed class ActivityLifecycle : UiEvent {
 
-  class Resumed(activityName: String?) : TheActivityLifecycle() {
+  class Resumed(activityName: String?) : ActivityLifecycle() {
     override val analyticsName = event(activityName, "Resumed")
   }
 
-  class Started(activityName: String?) : TheActivityLifecycle() {
+  class Started(activityName: String?) : ActivityLifecycle() {
```

Comment:
Move `Started` above `Resumed`? For the sake of order.

Reply:
Done.